tests/unit/test_util.py

```diff
@@ -15,186 +13,175 @@
 )
 
 
-class TestUtil(TestCase):
-    """
-    Test the functionality in util.py
-    """
-
-    def test_is_constant_and_can_evaluate(self):
-        symbol = pybamm.PrimaryBroadcast(0, "negative electrode")
-        self.assertEqual(False, pybamm.is_constant_and_can_evaluate(symbol))
-        symbol = pybamm.StateVector(slice(0, 1))
-        self.assertEqual(False, pybamm.is_constant_and_can_evaluate(symbol))
-        symbol = pybamm.Scalar(0)
-        self.assertEqual(True, pybamm.is_constant_and_can_evaluate(symbol))
-
-    def test_fuzzy_dict(self):
-        d = pybamm.FuzzyDict(
-            {
-                "test": 1,
-                "test2": 2,
-                "SEI current": 3,
-                "Lithium plating current": 4,
-                "A dimensional variable [m]": 5,
-                "Positive electrode diffusivity [m2.s-1]": 6,
-            }
+def test_is_constant_and_can_evaluate():
+    symbol = pybamm.PrimaryBroadcast(0, "negative electrode")
+    assert False is pybamm.is_constant_and_can_evaluate(symbol)
```

Comment:
```suggestion
    assert not pybamm.is_constant_and_can_evaluate(symbol)
```
I think this should be same but a bit simplified, make sure to test once tho.